carma/launch/plugins.launch.py

```diff
@@ -451,13 +454,40 @@ def generate_launch_description():
                 ],
                 parameters=[
                     vehicle_characteristics_param_file, #vehicle_response_lag
-                    pure_pursuit_tuning_parameters, #pure_pursuit calibration parameters
```

Comment:
Just wanted to check if this was intentional? I think this would still be needed for `pure_pursuit_wrapper`